flake.nix

```diff
@@ -16,6 +16,17 @@
         packages.default = naersk-lib.buildPackage {
           root = ./.;
           nativeBuildInputs = with pkgs; [ hidapi ];
+          postInstall = ''
+            # copy the manpages
+            install -D -m 0644 ${./.}/doc/nitrocli.1 $out/share/man/man1
+            # make completions
+            mkdir -p $out/share/bash-completion/completions/
+            cargo run --bin=shell-complete bash > $out/share/bash-completion/completions/nitrocli
+            mkdir -p $out/share/zsh/site-functions/
+            cargo run --bin=shell-complete zsh > $out/share/zsh/site-functions/_nitrocli
+            mkdir -p $out/share/fish/vendor_completions.d/
+            cargo run --bin=shell-complete fish > $out/share/fish/vendor_completions.d/nitrocli.fish
```

Comment:
Would you mind using long options when available? Makes it easier to understand the code.